string product_name

# request type: 
# 0=place in basket (for online order)
# 1=pick for customer
# 2=lead customer to product
uint8 request_type

# customer requests will have an order id of 0
uint32 order_id






